proximal/tests/test_algs.py

```diff
@@ -257,17 +253,16 @@ def test_half_quadratic_splitting_single_prox_fn(self):
                          max_iters=100,
                          max_inner_iters=50)
         self.assertItemsAlmostEqual(X.value, B, eps=2e-2)
-        self.assertAlmostEqual(sltn, 0)
```

Comment:
Can we keep ``self.assertAlmostEqual(sltn, 0)`` in the tests? It makes it clear what the solution is supposed to be.

Reply:
Happy to comply. Please refer to https://github.com/comp-imaging/ProxImaL/commit/e969c8bc4751d7e67c5df37ce253391243a3995c